code/game/objects/structures/droppod.dm

```diff
@@ -125,10 +127,9 @@ GLOBAL_LIST_INIT(blocked_droppod_tiles, typecacheof(list(/turf/open/space/transi
 	target_x = new_x
 	target_y = new_y
 	var/mob/notified_user = LAZYACCESS(buckled_mobs, 1)
-	if(notified_user)
-		. = checklanding(notified_user)
-		if(.)
-			balloon_alert(notified_user, "Coordinates updated")
+	. = checklanding(notified_user)
+	if(notified_user && .)
+		balloon_alert(notified_user, "Coordinates updated")
```

Comment:
???

Reply:
Pods can get coords set without a mob occupant, and we need to know if the target coords were successfully updated or not regardless of that.